OleksiiBozhchenko/HW4/Task1.py

```diff
@@ -0,0 +1,11 @@
+# Task1. Write a script that will calculate the factorial of the entered number without using recursion.
+
```

Comment:
good job


input_number = int(input("Input natural number: "))

fact = 1

if input_number == 1 or input_number == 0:
    print("Factorial {}! =".format(input_number), fact)
elif input_number < 0:
    print("Factorial non positive number don't exist")
else:
    for item in range(1,input_number + 1):
        fact *= item
    print("Factorial {}! =".format(input_number), fact)

Reply:
Thank You!
This kind of output from print() - new for me, I will study it.